source/manuals/logit-io.html.md.erb

```diff
@@ -266,24 +226,26 @@ Declare an action on matching messages that uses the `omfwd` module to forward t
 )
 ```
 
+```
 Note the `streamdriver="gtls"` and `streamdrivermode="1"` are required here,
 despite the earlier config fragment implying that the default behaviour has
 been changed.
+```
 
 ## Proxy patterns
 
-Architectural constraints around egress security often mean that having each of
-your servers connect directly to Logit's endpoint is not an option.
+Architectural constraints around Egress security often mean that having each of your servers connect directly to Logit's endpoint is not an option.
```

Comment:
Egress is a standard English word, not a brand name, so should be lower case as "egress" here.

Reply:
There is an Egress Software, but not relevant here.